_common/qiskit/execute.py

```diff
@@ -365,77 +371,31 @@ def set_execution_target(backend_id='qasm_simulator',
-                    
-                    service = QiskitRuntimeService()
-                    session_count += 1
-                    
-                    backend = service.backend(backend_id)
-                    session = Session(service=service, backend=backend_id)
-                    
-                    # get Sampler resilience level and transpiler optimization level from exec_options
-                    options = Options()
-                    # options.resilience_level = exec_options.get("resilience_level", 1)
-                    # options.optimization_level = exec_options.get("optimization_level", 3)
-                    
-                    # special handling for ibmq_qasm_simulator to set noise model
-                    if backend_id == "ibmq_qasm_simulator":
-                        this_noise = noise
-                        # get noise model from options; used only in simulator for now
-                        if "noise_model" in exec_options:
-                            this_noise = exec_options.get("noise_model", None)
-                            if verbose:
-                                print(f"... using custom noise model: {this_noise}")
-                        # attach to backend if not None
-                        if this_noise != None:
-                            options.simulator = {"noise_model": this_noise}
-                            metrics.QV = this_noise.QV
-                            if verbose:
-                                print(f"... setting noise model, QV={this_noise.QV} on {backend_id}")
-                        
-                    if verbose:
-                        print(f"... execute using Sampler on backend_id {backend_id} with options = {options}")
-                    
-                    # create the Qiskit Sampler with these options
-                    sampler = Sampler(session=session, options=options)
-                    
-                # otherwise, use provider and old style backend
-                # for IBM, create backend from IBMQ provider and given backend_id
-                else: 
-                    if verbose:
-                        print(f"... execute using Circuit Runner on backend_id {backend_id}")
-                        
-                    provider = IBMQ.get_provider(hub=hub, group=group, project=project)
-                    backend = provider.get_backend(backend_id)
-            else:
-                print(authentication_error_msg.format("IBMQ"))
-
         ###############################
         # otherwise, assume the backend_id is given only and assume it is IBM Cloud device
         else:
-            from qiskit_ibm_runtime import QiskitRuntimeService, Session, SamplerOptions as Options, SamplerV2 as Sampler
+            # need to import `Session` here to avoid the collision with
+            # `azure.quantum.job.session.Session`
+            from qiskit_ibm_runtime import Session
```

Comment:
`QiskitRuntimeService`, `SamplerOptions`and `SamplerV2` from `qiskit_ibm_runtime` are now imported at the beginning of the script.